packages/back-end/src/controllers/license.ts

```diff
@@ -28,10 +28,11 @@ import { updateSubscriptionInDb } from "../services/stripe";
  * want to restart their servers.
  */
 export async function getLicenseData(req: AuthRequest, res: Response) {
-  const context = getContextFromReq(req);
-
-  if (!context.permissions.canManageBilling()) {
-    context.permissions.throwPermissionError();
+  if (!req.superAdmin) {
```

Comment:
do we want all admins to view this or just superadmins?

Reply:
Only super admins, or those with billing permissions can see it (which is what normal admins have) will be able to use this endpoint.

Reply:
I don't think the superAdmin check is 100% necessary. `canManageBilling` ultimately calls `checkGlobalPermission` which automatically returns true if the user is a superAdmin.

Reply:
It is 100% necessary in the getContextFromReq is what fails if the superAdmin is not a member of the org.  If we are expecting superadmins to be able to call context then perhaps an exception needs to be made in that function.

Reply:
Ah - I see now. Disregard. :)